qt/aqt/browser/card_info.py

```diff
@@ -47,10 +64,87 @@ def _setup_ui(self, card_id: CardId) -> None:
         self.setLayout(layout)
 
         self.web.eval(
-            f"anki.cardInfo(document.getElementById('main'), {card_id}, true);"
+            "const cardInfo = anki.cardInfo(document.getElementById('main'));\n"
+            "cardInfo.then((c) => c.$set({ includeRevlog: true }));"
```

Comment:
I think it would be simpler to give includeRevlog a default value of true in the component, instead of defaulting to uninitialized, and requiring the caller to set it.